__tests__/app.test.js

```diff
@@ -52,28 +52,74 @@ describe('GET /api/articles/:article_id', () => {
         });
       });
   });
+});
+test('400: responds with an error message if id is not a valid type', () => {
+  return request(app)
+    .get('/api/articles/banana')
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe('bad request');
+    });
+});
+test('404: responds with an error message if article does not exist', () => {
+  return request(app)
+    .get('/api/articles/14')
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe('path not found');
+    });
+});
+
+describe('GET /api', () => {
+  test('200: returns an object with all the endpoints', () => {
+    return request(app)
+      .get('/api')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(endPoints);
+      });
+  });
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+  test('200: returns all comments in an array for an article id', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body);
+        body.comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe('number');
+          expect(typeof comment.votes).toBe('number');
+          expect(typeof comment.created_at).toBe('string');
+          expect(typeof comment.author).toBe('string');
+          expect(typeof comment.body).toBe('string');
+          expect(typeof comment.article_id).toBe('number');
```

Comment:
We know the article ID in this instance, so can be more specific with your expect statement 👍